cmd/monaco/integrationtest/cleanup.go

```diff
@@ -20,62 +20,44 @@ package integrationtest
 
 import (
 	"fmt"
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/integrationtest/utils/test"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/timeutils"
-	"github.com/stretchr/testify/assert"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
 	"path/filepath"
-	"strings"
 	"testing"
-
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
-	"github.com/spf13/afero"
 )
 
 // CleanupIntegrationTest deletes all configs that are defined in a test manifest. It uses the CLI runner, to call the
 // deletefile.Command to generate a delete file for the test manifest, and delete.GetDeleteCommand to remove configs using
 // the generated file.
-func CleanupIntegrationTest(t *testing.T, fs afero.Fs, manifestPath string, specificEnvironments []string, suffix string) {
-
-	var envArgs []string
-	if len(specificEnvironments) > 0 {
-		envArgs = []string{
-			"--environment",
-			strings.Join(specificEnvironments, ","),
-		}
+func CleanupIntegrationTest(t *testing.T, fs afero.Fs, manifestPath string, environment string, suffix string) {
+	var env string
+	if len(environment) > 0 {
+		env = fmt.Sprintf("--environment %s", environment)
 	}
 
-	log.Info("### Generating delete file for test cleanup ###")
-
 	deleteFile := fmt.Sprintf("delete-%s-%s.yaml", timeutils.TimeAnchor().UTC().Format("20060102-150405"), suffix)
 
 	absManifestPath, err := filepath.Abs(manifestPath)
-	assert.NoError(t, err)
-	cmd := runner.BuildCli(fs)
-	args := append([]string{
-		"generate",
-		"deletefile",
-		absManifestPath,
-		"--file", deleteFile,
-		"--exclude-types", "builtin:networkzones",
-	}, envArgs...)
-	cmd.SetArgs(args)
-	err = cmd.Execute()
-	assert.NoError(t, err)
-
-	log.Info("### Cleaning up after integration test ###")
-
-	cmd = runner.BuildCli(fs)
-	args = append([]string{
-		"delete",
-		"--manifest", manifestPath,
-		"--file", deleteFile,
-	}, envArgs...)
-	cmd.SetArgs(args)
-	err = cmd.Execute()
+	require.NoError(t, err)
+	{
+		_, err := test.Monacof(
+			"monaco generate deletefile %s --file %s --exclude-types builtin:networkzones %s", absManifestPath, deleteFile, env,
+		).WithFs(fs).Run()
+		require.NoError(t, err)
+		if df, err := filepath.Abs(deleteFile); err == nil {
+			if b, err := afero.ReadFile(fs, df); err == nil {
+				fmt.Println(string(b[:]))
+			}
+		}
+	}
 
-	if err != nil {
-		t.Log("Failed to cleanup all test configurations, manual/nightly cleanup needed.")
-	} else {
-		t.Log("Successfully cleaned up test configurations.")
+	{
+		_, err := test.Monacof("monaco delete --manifest %s --file %s %s", manifestPath, deleteFile, env).WithFs(fs).Run()
```

Comment:
why passing monaco as the first argument, and sometimes not?
Maybe we make it more strict, like always requiring the first word to be the command and use it always like this?
Otherwise it is unclear, and one would need to look down the code what actually happens to the monaco prefix.

Reply:
this - as commented on one of the other PRs that had these changes. I'd also just drop the monaco prefix entirely. It's currently only there for printing a "full" command including default CLI binary name, and code strips it out again anyway.

Reply:
With monaco it mimics the actual CLI command. Without it, it requires more internal knowledge of how things work. I found it helpful to have a `monaco` prefix when searching for a command.

My only argument for `monaco` prefix is that it looks nicer prefix :smiling_face_with_three_hearts: 

@arthurpitman , do you have some opinion on this
@UnseenWizzard , @warber Just make jour verdict, and I'll implement it.

Regarding showing/printing the command, it will always print monaco prefix. The idea is to print the exact command that is executed.

Reply:
I'm fine with having the `monaco` prefix - and even requiring it. I'm also fine if we drop it completely.
My only strong opinion is that I don't want it to sometimes have the prefix and sometimes not.

If I'd add a new test using this I'll look at what is there already, so having different ways it's used can be confusing.

Reply:
My preferred option would be with also a highly unique prefix so I can search for it even more easily that just with `monaco`. Personally, I rarely run things from the command line directly for testing but prefer launch configs set up in my IDE, but I do see the value in clearly seeing what command was run.

Reply:
`monaco` prefix is required than.